core/src/main/java/com/homeaway/streamingplatform/db/dao/impl/KafkaManagerImpl.java

```diff
@@ -72,7 +72,8 @@ public void upsertTopics(Collection<String> topics, int partitions, int replicat
             List<String> topicsToCreate = new ArrayList<>();
             for (String topic : topics) {
                 if (AdminUtils.topicExists(zkUtils, topic)) {
-                    updateTopic(zkUtils, topic, topicConfigMap);
+                    log.info("Stream Registry does not allow updating configs for existing topics.");
+                    //updateTopic(zkUtils, topic, topicConfigMap);
```

Comment:
I thought we were going to fix this upstream.

Reply:
I don't think we want to disable this functionality.

Reply:
@neoword  I am working on a change to check if the config is same as the existing topic config when moving to stream registry for the first time. I will update the PR and send it back again.